force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +125,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
+        Integer vowelCount = 0;
 
-        /*
-        for () {
+        // Convert the string to lower case to ignore case
+        str = str.toLowerCase();
+
+        // Loop through each character in the string
+        for (Integer i = 0; i < str.length(); i++) {
             // get individual characters from the string - hint use substring or split
+            String ch = str.substring(i, i + 1);
```

Comment:
You made this work with the substring() method which works, Have a look at the split() method, which makes this even easier.